### PR TITLE
using emailSender service

### PR DESCRIPTION
**Pull request by flovntp**

### CODE REVIEW

File: .environment [hunk 1]
```diff
@@ -9,4 +9,8 @@ export DATABASE_HOST=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".
 export DATABASE_PORT=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].port")
 export DATABASE_NAME=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].path")
 export DATABASE_USER=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].username")
-export DATABASE_PASSWORD=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].password")
\ No newline at end of file
+export DATABASE_PASSWORD=$(echo $PLATFORM_RELATIONSHIPS | base64 --decode | jq -r ".database[0].password")
+
+# data mailer sender
+MAILER_SENDER=devrel@internal.platform.sh
```
Comment (flovntp) [inline .environment:15]: i don't know if that's secure to let this email address appears here, or if we should use a P.sh envVar (level=environment) for that

Comment (thomasdiluccio) [reply .environment:15]: I'd use a PSH env var.

File: LOCAL_DEV.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+To run this project locally, you need DDEV (and so, Docker) and then:
+
+```bash
+ddev start
+ddev ssh
+composer install 
+npm run dev 
+php bin/console d:s:u --dump-sql --force
+php bin/console d:f:l -e dev
```
Comment (thomasdiluccio) [inline LOCAL_DEV.md:9]: We should run those commands from the DDEV container.

```suggestion
ddev sf d:s:u --dump-sql --force
ddev sf d:f:l -e dev
```

**Review by thomasdiluccio — APPROVED** [review]

File: LOCAL_DEV.md (hunk 2 above)
Comment (thomasdiluccio) [reply LOCAL_DEV.md:9]: ignore this. I skipped the ddev ssh.

Comment (flovntp) [reply LOCAL_DEV.md]: i miss the `ddev sf` command availability, so i change to use it, you're right

**Pull request merged**